### ISSUE TITLE
Question about the estimated people vaccinated model.

### ISSUE THREAD

**Issue by timostrating** [Sikerdebaard/opencoronastats#4]
First of all thanks for making this. I look at the website daily after found it on Reddit.

I have a small question about this graph: https://covid-analytics.nl/vaccination.html#chart-estimated-people-vaccinated
![image](https://user-images.githubusercontent.com/12185305/108623697-a3d05300-7440-11eb-998f-c82c2e330ff9.png)

As far as I understand the graph is based on a csv file from Our World In Data. And that csv is again based on the dashboard from the dutch government. https://github.com/Sikerdebaard/opencoronastats/blob/master/scripts/vaccine.py#L19

But because that dashboard does not specify if a vaccin is used as a first or second portion other data is added to a Model to separate this in 2 separate graphs.
https://github.com/Sikerdebaard/opencoronastats/blob/master/scripts/vaccine.py#L73

The part that I do not understand is that https://www.rivm.nl/covid-19-vaccinatie/cijfers-vaccinatieprogramma makes the estimate that 154_445 have been vaccinated for the second time.
![image](https://user-images.githubusercontent.com/12185305/108623905-dcbcf780-7441-11eb-83d7-4a2026be145b.png)

How does the model then get this green graph?

The first dutch person to get the vaccin for the second time was on the 27 of januari https://www.telegraaf.nl/nieuws/1505464797/eerste-nederlanders-krijgen-tweede-prik So that makes me think fully vaccinated means first + second vaccin and some weeks to make the the body adapt and learn. 

So in summary:
1) could you please elaborate what fully vaccinated means: 2 vaccines or 2 vaccines + some time for the body to adapt or something else
2) How does "A model based approuch" mean in this context?
3) The graph is currently going down is this just data that will be altered later is is the data from the green graph altering the orange graph.
4) The OWID scv file gets combined with data from a scraper that scrapes these json files from the dashboard https://coronadashboard.rijksoverheid.nl/json/NL.json why is there the need to combine those two?

**Comment by Sikerdebaard** [comment]
Hi @timostrating, thanks for your ticket!

Unfortunately a bug caused our model to report incorrect data regarding vaccinations in the Netherlands starting yesterday evening around 20:00. We have since corrected this bug and are now reporting more sensible numbers again. As such, please ignore the chart in your post as this data is erroneous and use an updated chart from our website instead.

**"could you please elaborate what fully vaccinated means: 2 vaccines or 2 vaccines + some time for the body to adapt or something else"**
Fully vaccinated means that this person has received all doses of a vaccine regimen. For example if John receives a single dose of a comirnaty vaccine on day 1 then he is counted as 'started a vaccine regimen'. If John then receives his second dose on day 21 he is then counted as 'fully vaccinated'. The model does not take extra time into account until the vaccine start working optimally as we currently do not have such data.  

**"How does "A model based approuch" mean in this context?"**
We are working on a computer model for estimating the number of people vaccinated on certain model constraints. E.g. what vaccines are in use by a specific country, what dose interval does the country use per vaccine and how many doses have been administered. You can find the models source code here. https://github.com/Sikerdebaard/global-vaccine-model

**"The graph is currently going down is this just data that will be altered later is is the data from the green graph altering the orange graph."**
We expect the orange line to move under the green line eventually. This is because people that have started a vaccine regimen will eventually be moved to the fully vaccinated group.

**"The OWID scv file gets combined with data from a scraper that scrapes these json files from the dashboard https://coronadashboard.rijksoverheid.nl/json/NL.json why is there the need to combine those two?"**
Since yesterday OWID is using our data-mirror of the gov. coronadashboard and we are now using the scraped data directly.


Did you know that our models estimates are available under a Creative Commons License over here? You can find the models output here: https://github.com/Sikerdebaard/opencoronastats/tree/master/data/vaccine_estimate

**Comment by timostrating** [comment]
Thanks for the quick reply, The new graph looks really nice. and thanks for all the explanations. 
![image](https://user-images.githubusercontent.com/12185305/108630186-d2abf080-7463-11eb-84ee-e25ac51e5e51.png)
